src/main/java/org/opentripplanner/middleware/auth/Auth0Connection.java

```diff
@@ -64,6 +64,12 @@ public static void checkUser(Request req) {
         }
     }
 
+    public static boolean isAuthHeaderPresent(Request req) {
+        final String authHeader = req.headers("Authorization");
+        return (authHeader != null) ? true : false;
+    }
```

Comment:
This method should also be used in `getTokenFromRequest`. Also, the return statement can be simplified to 
```java
return authHeader != null;
```

Reply:
Updated.

Reply:
Not a big deal, but notice that the header is retrieved again two lines after this function call...